src/webview-src/redesign.css

```
@@ -0,0 +1,1032 @@
+  align-items: center;
+  justify-content: center;
+  width: 36px;
+  height: 36px;
+  padding: 0;
+  background: var(--oh-bg-interactive);
+  border: 1px solid rgba(255, 225, 101, 0.2);
+  border-radius: var(--oh-radius-md);
+  color: var(--oh-text-primary);
+  cursor: pointer;
+  transition: all var(--oh-transition-base);
+  position: relative;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

position is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)